flags/flags.go

```diff
@@ -7,7 +7,6 @@ const (
 	AppID         = "app-id"
 	URL           = "url"
 	Version       = "version"
-	Create        = "create"
```

Comment:
@o-volkov - if we are removing this flag, needs to get removed from below:
- https://github.com/ketch-com/ketch-cli/blob/4c284d087bc8dd2cb58a4783bc04f12e56ba7e91/action.yml#L19
- https://github.com/ketch-apps/alchemer/blob/c49ddcb25b5b9c93f95bea41690ad05c3d22492a/.github/workflows/ci.yml#L44    x 14 other apps

Reply:
@sethyates - are we removing the ability to separately `create` and `publish` an app via ketch-cli? Seems like that is what is happening here with removal of `--create` flag.

From what I could gather, apps are separately `create`d and `publish`ed because anyone can create an app but only those that are published can be installed by others in the marketplace. Are we removing that distinction?

Reply:
@sethyates - are we removing the ability to separately create and publish an app via ketch-cli? Seems like that is what is happening here with removal of --create flag.

From what I could gather, apps are separately created and published because anyone can create an app but only those that are published can be installed by others in the marketplace. Are we removing that distinction?

Reply:
This flag was only used to validate app ID (if --create is false, then app ID is required).  App publishing (sending for moderation) always happens, regardless of this flag

Reply:
But previously, the flag would signal 1) if the app should be published only (no create flag) or 2) both created then published (with --create flag). Removing the flag would remove ability to do 1) publish only. If that was the intention no problem, just wanted to confirm we expect `ketch-cli` to only do 2) create+publish only now (not both 1) publish only and 2) create + publish).

Latest in main is slightly different than what is in the `sethyates/feat/feat-ftw` branch if that clarifies at all: https://github.com/ketch-com/ketch-cli/blob/4c284d087bc8dd2cb58a4783bc04f12e56ba7e91/apps/publish.go#L102

Reply:
@smalexan and I discussed - we can just get rid of the `--create` flag as you have it here, since it's confusing anyway and we won't have a use case where we need to use the `ketch-cli` to `publish` only since there is no way currently to `create` only